### PR TITLE
Quick bump to 2.1.20

### PR DESCRIPTION
**Pull request by jurf**
Skipping the AppData/SDK changes for now due to lack of time.

### CODE REVIEW

File: net.ankiweb.Anki.yaml [hunk 1]
```diff
@@ -178,8 +178,8 @@ modules:
       - ln -s /app/share/locale /app/share/anki/locale
     sources:
       - type: archive
-        url: https://apps.ankiweb.net/downloads/current/anki-2.1.15-source.tgz
-        sha256: 5a53760164c77d619f55107a13099cffe620566a7f610b61b6c4b52487f3bb89
+        url: https://github.com/ankitects/anki/archive/2.1.20.tar.gz
+        sha256: 04b7bc7302a2e3f12b7d077d1ccedef08922ae189c659d81fd8e22456fd30d45
```
Comment (filalex77) [inline net.ankiweb.Anki.yaml:182]: Anki 2.1.25 [has been released](https://changes.ankiweb.net/#/?id=changes-in-2125).

```suggestion
        url: https://github.com/ankitects/anki/archive/2.1.25.tar.gz
        sha256: b19c91c957055d25a91abebd6c61f53d6e23ac972d39639c54465918440fa853
```

Comment (jurf) [reply net.ankiweb.Anki.yaml:182]: I know. It has a horrible build system however, I just haven’t found the time to make it work with offline building in Flatpak. Not mentioning the hours I killed on trying to build PyQtWebEngine on Qt 5.14, that still isn’t working. :|

Comment (filalex77) [reply net.ankiweb.Anki.yaml:182]: Okay, that sounds... Hard. Well, no need to rush with this update then. Thanks for your work!

**Comment by jurf** [comment]
Closed in favour of #32.

**Pull request closed without merging**